index.html

```diff
@@ -221,20 +221,20 @@ <h2 class="foot-section-h">Get started</h2>
 
 
     <form action="https://formspree.io/f/xaykaovn" method="post">
-        <div class="your-name-container" >
+		<div class="your-name-container" >
             <div>Your name</div>
-            <input type="text" name="your name" placeholder="Anastasiia" maxlength='30' required id="">
+            <input type="text" name="" placeholder="Anastasiia" maxlength='30'  id="">
         </div>
 
 
         <div class="email-container">
             <div>Your email</div>
-            <input type="email" placeholder="|"  name="your email" id="" required >
+            <input type="email" placeholder="|"  name="your email" id=""  >
         </div>    
 
-         <textarea class="form-coment" name="form-coment" id="" placeholder="Enter your message here" maxlength='500' required cols="30" rows="10"></textarea>
-        
-        <button class="start-coll-btn" type="submit" >Start collaboration</button>
+         <textarea class="form-coment" name="form-coment" id="" placeholder="Enter your message here" ></textarea>
```

Comment:
- Hey, please, kindly update the text-area to the previous way it was adding this missing part of the validation `maxlength='500' required cols="30" rows="10"`